helm-chart-sources/pulsar/values.yaml

```diff
@@ -1501,9 +1501,9 @@ zoonavigator:
       cpu: 0.1
   image:
     repository:
-      web: elkozmon/zoonavigator-web
-      api: elkozmon/zoonavigator-api
-    tag: 0.6.0
+      web: elkozmon/zoonavigator
+      api: elkozmon/zoonavigator
+    tag: latest
```

Comment:
Since users can override this image tag manually in their values files, I think we should specify an actual version instead of `latest` here.